src/models/Xlsx.js

```diff
@@ -0,0 +1,21 @@
+var mongoose = require("mongoose");
+const { ObjectId } = mongoose.Schema;
+const mongoosePaginate = require("mongoose-paginate-v2");
+
+const xlsxSchema = new mongoose.Schema(
+  {
+    xlsxUrl: {
+      type: String,
+      required: true,
+    },
+
+    // classsequence: {
```

Comment:
@adityaVinayak each excel upload should have a batch process id (do not use the mongo generated id for display, as it is very lengthy). you can generate a sequential `batch process id`.

Reply:
done